### PR TITLE
Filtering and formatting

### PR DESCRIPTION
**Pull request by mixalturek**

### CODE REVIEW

File: filter/src/main/java/com/avast/metrics/filter/ConfigLoader.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    private static final String ENABLED_SUFFIX = CONFIG_NAME_SEPARATOR + "enabled";
+
+    private final String nameSeparator;
+
+    ConfigLoader(String nameSeparator) {
+        this.nameSeparator = nameSeparator;
+    }
+
+    List<FilterConfig> load(Config config) {
+        Config referenceConfig = ConfigFactory.defaultReference().getConfig(SECTION_DEFAULTS);
```
Comment (jendakol) [inline filter/src/main/java/com/avast/metrics/filter/ConfigLoader.java]: Make this static.

Comment (mixalturek) [inline filter/src/main/java/com/avast/metrics/filter/ConfigLoader.java]: Public API is `MetricsFilter.fromConfig()` which is static.

Comment (jendakol) [inline filter/src/main/java/com/avast/metrics/filter/ConfigLoader.java]: I'm talking about the `referenceConfig` field :-)

Comment (mixalturek) [inline filter/src/main/java/com/avast/metrics/filter/ConfigLoader.java]: Aaah. Ehm, why? It would only waste memory, very close to memory leak.

**Pull request merged**